src/modules/coiffeur/events/coiffeur-message.listener.ts

```diff
@@ -0,0 +1,24 @@
+import { Message } from 'discord.js';
+import {
+  WHAT,
+  WHAT_ANSWER,
+  WHY,
+  WHY_ANWSER,
+} from '@/modules/coiffeur/coiffeur-constants.js';
+
+export async function coiffeurMessageListener(message: Message): Promise<void> {
+  if (message.author.bot) return;
+
+  const content = message.content.toLowerCase();
+  const shouldReply = Math.random() > 0.66;
```

Comment:
Peut-être séparer cette expression dans une méthode qui détermine si le coiffeur doit répondre ou pas à l'utilisateur, pour prévoir de futures conditions au besoin, et aussi documenter la probabilité que tu as utilisé (là comme ça c'est pas super intuitif, c'est 34% de chances je pense si je comprends bien, mais c'est pas rapide à voir du coup

Reply:
```suggestion
  const shouldReply = shouldReply(message);
```

En gros tu appelles une méthode qui s'occupe de te dire si oui ou non le bot doit répondre à partir d'un message, tu peux mettre la méthode où tu veux, tu peux la garder dans le même fichier vu qu'elle va être très simple:
```ts
function shouldReply(message){
  // toujours à modifier bien sûr
  return Math.random() > 0.66;
}
```